CSS/Components/images.css

```diff
@@ -0,0 +1,17 @@
+.round-img {
+  border-radius: 50%;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
+
+.square-img {
+  border-radius: 5px;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
```

Comment:
you can create a separate img class for defining repeating properties
such as:
`.images{
  width: 200px;
  height: 200px;
  object-fit: cover;
}`

Reply:
Thanks, will look into that